optimum/exporters/neuron/convert.py

```diff
@@ -174,7 +179,7 @@ def export_neuronx(
         output (`Path`):
             Directory to store the exported Neuron model.
         auto_cast (`optional[str]`, defaults to `"none"`):
```

Comment:
```suggestion
        auto_cast (`Optional[str]`, defaults to `"none"`):
```
Still counterintuitive here (`"none"` vs `None`)

Reply:
The default is set to None now.